code/rdb/datastripe.q

```diff
@@ -10,7 +10,9 @@
     .lg.o[`reload;"Kept ",string[.ds.periodstokeep]," period",$[.ds.periodstokeep>1;"s";""]," of data from : ",", " sv string[tabs]];
 
     // update the access table in the rdb
-    .rdb.access:update start:lasttime^(.ds.getstarttime each key .rdb.tablekeycols), stptime:data[][`time] from .rdb.access;
+    t:tables[`.] except .rdb.ignorelist;
+    starttimes:.ds.getstarttime each t;
+    .rdb.access:update start:starttimes, stptime:data[][`time] from .rdb.access;
```

Comment:
is it guaranteed that `t` will be in the right order? I assume `.rdb.access` includes the table name, can this not just be something like

```suggestion
    .rdb.access:update start:.ds.getstarttime each table, stptime:data[][`time] from .rdb.access;
```

(assuming table name column in `.rdb.access` is `table`)

Reply:
Yes, you are right!